.github/workflows/merge.yml

```diff
@@ -21,4 +21,5 @@ jobs:
       
       - name: docker build
         run: |
-          make build-docker gcr-repo=${{ secrets.RELEASE_REPO }}
+          make build-docker-ig tag=${{ github.sha }}
```

Comment:
This will build only Ig, that's correct?

Reply:
It won't be better something like below in the make file:
`docker build docker/7.0/ig/ -t eu.gcr.io/${gcr-repo}/securebanking/gate/ig:latest -t eu.gcr.io/${gcr-repo}/securebanking/gate/ig:${tag}`
and
`docker push ....`

And call the make file only once `make build-docker-ig tag=xxx gcr-repo=xxx`